src/styles/_Checkout.scss

```diff
@@ -0,0 +1,62 @@
+@use 'variables' as v;
+
+@mixin space-between {
```

Comment:
Haven't done it yet, but I think mixins can go in the _mixins.scss file and be called like variables.